staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_clientset.go

```diff
@@ -134,9 +134,15 @@ func (c *Clientset) Discovery() $.DiscoveryInterface|raw$ {
 
 var newClientsetForConfigTemplate = `
 // NewForConfig creates a new Clientset for the given config.
+// If config's RateLimiter is not set and QPS and Burst are acceptable, 
+// NewForConfig will generate a rate limit in config, and the rate 
+// limit will be shared by other clients from same config.
```

Comment:
sorry... just noticed this comment... the rate limiter is set in configShallowCopy, not in config, so other clients created from the same config will not share the ratelimiter, right?

Reply:
yes, I made a mistake,